.travis.yml

```diff
@@ -1,19 +0,0 @@
-language: python
-
-python:
-  - 3.5
-  - 3.6
-  - 3.7
-
-install:
-  - source prepareenv.sh
-  - source .github/travis/check_license.sh
-  - source .github/travis/check_python_script.sh
```

Comment:
Didn't move over license and check python scripts?

Reply:
Fixed.